coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -197,15 +225,16 @@ def __init__(self):
                                         'GITHUB_REPO_BRANCH',
                                         DEFAULT_GITHUB_REPO_BRANCH)
         self.koji_user         = COREOS_KOJI_USER
+        self.koji_client       = koji.ClientSession(KOJI_SERVER_URL)
 
-        # If a keytab was specified let's use it
+        # If a keytab was specified let's try to auth.
         self.keytab_file = os.getenv('COREOS_KOJI_TAGGER_KEYTAB_FILE')
+        if self.keytab_file and not os.path.exists(self.keytab_file):
```

Comment:
Any reason to not just embed this check inside the following `if` block? e.x.:

```
if self.keytab_file:
  if not os.path.exists(self.keytab_file):
    raise Exception("...")
  principal = ...
```

Reply:
i can merge them